task24/css/style.css

```diff
@@ -0,0 +1,114 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.loader,
+.loader:before,
+.loader:after {
+  background: #ccc;
+  -webkit-animation: load1 1s infinite ease-in-out;
+  animation: load1 1s infinite ease-in-out;
+  width: 1em;
+  height: 4em;
+}
+.loader {
+  color: #ccc;
+  text-indent: -9999em;
+  margin: 88px auto;
+  position: relative;
+  font-size: 11px;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation-delay: -0.16s;
+  animation-delay: -0.16s;
+}
+.loader:before,
+.loader:after {
+  position: absolute;
+  top: 0;
+  content: "";
+}
+.loader:before {
+  left: -1.5em;
+  -webkit-animation-delay: -0.32s;
+  animation-delay: -0.32s;
+}
+.loader:after {
+  left: 1.5em;
+}
+@-webkit-keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+@keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+
+/* .modal-wrapper {
+  display: none;
+} */
+
+/* .modal-wrapper {
+  position: fixed;
+  inset: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 50;
+  background-color: rgba(0, 0, 0, 0.7);
+}
+.modal {
+  max-width: 600px;
+  height: 400px;
+  padding: 16px;
+  background-color: white;
+  overflow: scroll; 
+} */
+
+.hover-item {
+  text-decoration: none;
+}
+
+.hover-text {
+  position: relative;
+  z-index: 60;
+}
```

Comment:
nits
Personally, I thought it would be better to add a pointer here.

``
cursor: pointer;
``

Reply:
You are right. Fixed!
[b47212d](https://github.com/seeeen044/JavaScript_handson/pull/26/commits/b47212df3d9a5eb755b084fde22dcfadd1a733dd)